### ISSUE TITLE
[Testing] Finding the Accuracy Performance by varying parameters in YOLOv5 model

### ISSUE THREAD

**Issue by jaskiratsingh2000** [jaskiratsingh2000/Research-Scalable-Vehicle-Detection-on-Edge-Devices#14]
The [yolov5s.yaml](https://github.com/ultralytics/yolov5/blob/master/models/yolov5s.yaml) configuration file inside the ultralytics/yolov5 repo contains various scaling parameters. Hence I scaled the **depth_multiple** and **width_multiple** values.

**Default Values:**

```
depth_multiple: 0.33
width_multiople: 0.50
```

**Scaled Values:**

```
depth_multiple: 0.01
width_multiople: 0.02
```

**Comment by jaskiratsingh2000** [comment]
latency received for this model **with scaled parameters** is **179.4ms total**. Results are the following:

```
                 from  n    params  module                                  arguments                     
  0                -1  1       880  models.common.Focus                     [3, 8, 3]                     
  1                -1  1       592  models.common.Conv                      [8, 8, 3, 2]                  
  2                -1  1       336  models.common.C3                        [8, 8, 1]                     
  3                -1  1       592  models.common.Conv                      [8, 8, 3, 2]                  
  4                -1  1       336  models.common.C3                        [8, 8, 1]                     
  5                -1  1      1184  models.common.Conv                      [8, 16, 3, 2]                 
  6                -1  1      1248  models.common.C3                        [16, 16, 1]                   
  7                -1  1      3504  models.common.Conv                      [16, 24, 3, 2]                
  8                -1  1      1512  models.common.SPP                       [24, 24, [5, 9, 13]]          
  9                -1  1      2736  models.common.C3                        [24, 24, 1, False]            
 10                -1  1       416  models.common.Conv                      [24, 16, 1, 1]                
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 12           [-1, 6]  1         0  models.common.Concat                    [1]                           
 13                -1  1      1504  models.common.C3                        [32, 16, 1, False]            
 14                -1  1       144  models.common.Conv                      [16, 8, 1, 1]                 
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 16           [-1, 4]  1         0  models.common.Concat                    [1]                           
 17                -1  1       400  models.common.C3                        [16, 8, 1, False]             
 18                -1  1       592  models.common.Conv                      [8, 8, 3, 2]                  
 19          [-1, 14]  1         0  models.common.Concat                    [1]                           
 20                -1  1      1248  models.common.C3                        [16, 16, 1, False]            
 21                -1  1      2336  models.common.Conv                      [16, 16, 3, 2]                
 22          [-1, 10]  1         0  models.common.Concat                    [1]                           
 23                -1  1      2928  models.common.C3                        [32, 24, 1, False]            
 24      [17, 20, 23]  1      1071  Detect                                  [2, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [8, 16, 24]]
Model Summary: 247 layers, 23559 parameters, 23559 gradients

 time (ms)     GFLOPs     params  module
     33.39       0.00        880  models.common.Focus
     11.78       0.00        592  models.common.Conv
     23.08       0.00        336  models.common.C3
      3.22       0.00        592  models.common.Conv
      9.14       0.00        336  models.common.C3
      8.02       0.00       1184  models.common.Conv
     21.11       0.00       1248  models.common.C3
      4.40       0.00       3504  models.common.Conv
     15.80       0.00       1512  models.common.SPP
     11.04       0.00       2736  models.common.C3
      1.42       0.00        416  models.common.Conv
      0.14       0.00          0  torch.nn.modules.upsampling.Upsample
      0.07       0.00          0  models.common.Concat
      6.56       0.00       1504  models.common.C3
      1.37       0.00        144  models.common.Conv
      0.19       0.00          0  torch.nn.modules.upsampling.Upsample
      0.10       0.00          0  models.common.Concat
      6.76       0.00        400  models.common.C3
      1.67       0.00        592  models.common.Conv
      0.07       0.00          0  models.common.Concat
      5.03       0.00       1248  models.common.C3
      1.60       0.00       2336  models.common.Conv
      0.06       0.00          0  models.common.Concat
      4.43       0.00       2928  models.common.C3
      8.97       0.00       1071  Detect
179.4ms total
```

**Comment by jaskiratsingh2000** [comment]
Performance Accuracy Results can be found here:

```
Fusing layers... 
Model Summary: 224 layers, 7266973 parameters, 0 gradients
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
val: Scanning 'valid/labels.cache' images and labels... 29 found, 0 missing
               Class     Images     Labels          P          R     mAP@.5
                 all         29          0          0          0          0          0
Speed: 14.0ms pre-process, 2151.7ms inference, 5.6ms NMS per image at shape (32, 3, 640, 640)
```
